package.json

```diff
@@ -33,7 +36,6 @@
     "@docusaurus/plugin-content-docs": "2.0.0-beta.17",
     "@docusaurus/preset-classic": "2.0.0-beta.17",
     "@docusaurus/types": "^2.0.0-beta.17",
-    "@easyops-cn/docusaurus-search-local": "file:third_party/docusaurus-search-local",
```

Comment:
let's keep the third_party folder in the repo for now so we could revert back easier if we are not happy.